R/zzz.R

```diff
@@ -1,10 +1,15 @@
-.onLoad <- function(libname, pkgname){
-
-  assign("stage", "production", envir = parent.env(environment()))
-  assign("base_url",
-         "https://equity-tool-api.urban.org",
-         envir = parent.env(environment()))
+.onAttach <- function(libname, pkgname) {
+  stage <- "production"
+  assign("stage", stage, envir=as.environment("package:sedtR"))
+  packageStartupMessage(paste0("Loading sedtR - using the ", stage, " API"))
```

Comment:
I think I just updated whatever was in place already for the .onAttach code. Did you want to remove the package start-up message entirely? I can't edit this branch so you or @alenastern will need to make the changes directly.

Reply:
fair point @Deckart2 - I just removed the startup message entirely. I've kept the assignment of the stage variable because we use it in some of our testing scripts